sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -192,6 +192,7 @@ private[hive] class IsolatedClientLoader(
     (name.startsWith("com.google") && !name.startsWith("com.google.cloud")) ||
     name.startsWith("java.lang.") ||
     name.startsWith("java.net") ||
+    name.startsWith("org.apache.derby.") ||
```

Comment:
This breaks Livy, as @HyukjinKwon pointed out. https://github.com/apache/spark/pull/20944#issuecomment-436530497